src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    book: bookReducer,
+  },
+});
```

Comment:
- [ ] You are required to import the categories reducer and set it up in the store as well, just as you have done with the books reducer.